src/components/ComboboxMenu.tsx

```diff
@@ -18,6 +20,7 @@ type ElementProps<MenuItemValue extends string | number, MenuGroupValue extends
   inputPlaceholder?: string;
   slotEmpty?: ReactNode;
   withSearch?: boolean;
+  useSearchInputComponent?: boolean;
```

Comment:
nit: prefix with something other than `use` since that's the convention for hooks